roles/jwflory.teleirc/tasks/compile.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: install go and build dependencies
+  package:
+    state: present
+    name:
+      - git
+      - golang
+
+- name: git clone/pull RITlug/teleirc
+  git:
+    repo: "https://github.com/RITlug/teleirc.git"
+    dest: "/tmp/teleirc"
+    version: HEAD
+    accept_hostkey: yes
+    force: yes
+
+- name: build teleirc binary
+  command:
+    cmd: bash build_binary.sh
+    chdir: "/tmp/teleirc/"
+
+- name: push teleirc binary to /usr/local/bin
+  copy:
+    remote_src: true
```

Comment:
I am not familiar with Ansible, but this was required for some reason in order see that the `/tmp/teleirc/teleirc` existed.